### PR TITLE
Preview to Main: Adding Video Review Promo Page

### PR DESCRIPTION
**Pull request by george-icarcover**

### CODE REVIEW

File: src/app/(main)/video_review_program/page.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+              </p>
+              <p className="pt-1.5 text-[16px] leading-[15px] min-[621px]:text-[20px] min-[621px]:leading-[22px] ">
+                Send them to us and get cash <LgBr /> via Zelle, Paypal, Venmo,
+                etc.
+              </p>
+            </div>
+          </div>
+        </section>
+        <section className="mb-[50px]  flex  flex-col bg-[black] px-4 pt-10  ">
+          <div className="flex flex-col px-8 text-[18px]  leading-[30px] min-[621px]:self-center min-[621px]:justify-self-center  min-[621px]:text-[20px]">
```
Comment (devwilliamy) [inline src/app/(main)/video_review_program/page.tsx:173]: Something I don't really check is the CSS, but something I've been seeing a lot is repeated flex.
Something I'd encourage you to keep in mind for the future is to see whether you actually do need the extra CSS class or not.

I'm like 90% sure you don't need the flex flex-col in the next line but I could be wrong

**Comment by devwilliamy** [comment]
Can you also make the .png into .webp?

**Comment by devwilliamy** [comment]
Is there supposed to be a link that links to the promo program?

Also can you change the name to `video-review-program` to be consistent with our other naming conventions for URL paths?

File: src/app/(main)/video_review_program/page.tsx (hunk 1 above)
Comment (george-icarcover) [reply src/app/(main)/video_review_program/page.tsx:173]: Flex-col is on the section to allow the inner div to be able to center itself (once it goes past 621px), without having to center everything inside the section (specifically the "participating will: ")

Comment (george-icarcover) [reply src/app/(main)/video_review_program/page.tsx:173]: I think I just forgot to take the inner flex flex-col out when I was taking out a gap-[ ]

**Pull request merged**